requirements.txt

```diff
@@ -1,4 +1,5 @@
 dwave-ocean-sdk>=3.3.0
-scikit-learn==1.0
+scikit-learn==1.0; python_version>='3.7'
+scikit-learn==0.24.2; python_version=='3.6'
```

Comment:
Oh, `scikit-learn` dropped py36 support in 1.0; makes sense. We also plan to drop it before its EOL in Dec this year.

Regardless, given the example works with a wide range of versions, how about just upper-bounding it? To `<=1.0`, or perhaps even to `<2.0`, in a hope they follow semver and won't break API until 2.0...